src/utils/clamp.ts

```diff
@@ -37,3 +37,6 @@ function clamp255(value: number): number {
 function clamp65535(value: number): number {
   return Math.min(Math.max(value, 0), 65535);
 }
+
+export const clamp = (min: number, max: number) => (value: number) =>
+  Math.max(Math.min(value, max), min);
```

Comment:
Please use regular named functions here. Also please rename the exported one so it's clear that it returns a function and not a clamped value.
Btw, any reason for using the opposite order of Math functions compared to the above implementations?

Reply:
What do you mean by "regular named functions"?

I used this syntax, which is apparently called "currying" because I saw it online and found it interesting. Do you think it's relevant to keep it or should I use a more standard syntax?

No, there is no reason for the order of the math functions, I just didn't pay attention.

Reply:
I mean something like this:

```ts
export function getClampFromTo(min: number, max: number) {
  return function clamp(value: number) {
    return Math.max(Math.min(value, max), min);
  };
}
```

This is still called currying (which is not a syntax). The syntax that you used is "arrow functions", which are fine as small callbacks for example in `array.map` but should be avoided in exported places or especially when returned from other functions because they are anonymous and this is always annoying in stack traces or when debugging.

Reply:
> Does it close any issues?

Yes, #176